src/main/java/spoon/decompiler/SpoonClassFileTransformer.java

```diff
@@ -136,6 +133,7 @@ public SpoonClassFileTransformer(Predicate<String> classNameFilter,
 			ProtectionDomain protectionDomain,
 			byte[] classfileBuffer
 	) throws IllegalClassFormatException {
+		IncrementalLauncher launcher = new IncrementalLauncher(inputSources, classPath, cache);
```

Comment:
I think it is better to build the model only after filtering classes the user does not want to transform. (And after decompilation has been tried with succes, althought this last one is less important).

Reply:
If it is not a low hanging fruit, I leave it as if for now.